src/main/java/frc/robot/subsystems/ShooterSubsystem

```diff
@@ -0,0 +1,119 @@
+// INTAKE one talon for feeding one neo for  pivot
```

Comment:
this should be ShooterSubsystem.java as the title, construct the class in robot container